File: lib/src/services/audio/tts_config.dart
/// Locked voice configuration for all TTS-generated audio (Type A, B, and C).
/// Use these parameters in every TTS API call, including batch generation.
class TtsConfig {
  TtsConfig._();

  /// Model: pin for product parity with batch tooling. Runtime Type C calls use
  /// `gemini-2.5-pro-preview-tts` on the Gemini API (see `functions/src/tts-config.ts`).
  static const String model = 'gemini-2.5-pro-tts';

  /// Voice name for Gemini Pro TTS.
  static const String voice = 'Erinome';

  /// Language code.
  static const String language = 'en-GB';

  /// Include this prompt in every TTS API call.
  static const String prompt =
      'Speak as an encouraging female British personal trainer. '
      'Use a natural, mild British accent. Warm and friendly. Clear and confident.';
}

File: functions/src/tts-config.ts
/**
 * Must stay aligned with lib/src/services/audio/tts_config.dart (Type C / batch TTS policy).
 */
export const TTS_MODEL = "gemini-2.5-pro-preview-tts";

/** Prebuilt voice name for Gemini Pro TTS (see Gemini speech-generation voice list). */
export const TTS_VOICE = "Erinome";

export const TTS_STYLE_PROMPT =
  "Speak as an encouraging female British personal trainer. " +
  "Use a natural, mild British accent. Warm and friendly. Clear and confident. " +
  "Conversational and relaxed — natural pacing, not overly formal or precisely enunciated.";
